session_redis/session.py

```diff
@@ -1,9 +1,10 @@
 # Copyright 2016-2024 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
 
+import builtins
```

Comment:
Maybe I missed something, but why you need to import `builtins` here?

You could use `list` directly.

If there's a name conflict (I didn't check), in this case, declare `list_type = list` at the start of the file.

Reply:
Added automatically by the precommit